py3status/modules/uname.py

```diff
@@ -26,7 +26,7 @@ class Py3status:
     """
     """
     # available configuration parameters
-    format = '{system} {release} {machine}'
```

Comment:
I am okay to remove `machine`

Reply:
I urge `system` too for QA.

Reply:
what QA

Reply:
QA in a sense where users does not have to create a `format` if they are trying to keep things simple and small (eg, they only want kernel version).

Reply:
I'd rather stick to this for now

Reply:
Pushing out `system`. Keep an eye on this!

Reply:
nop, leave system alone plz

Reply:
Yes. I bring it back. It's ready now.